sql/hive/src/test/scala/org/apache/spark/sql/sources/BucketedWriteSuite.scala

```diff
@@ -59,11 +59,18 @@ class BucketedWriteSuite extends QueryTest with SQLTestUtils with TestHiveSingle
     intercept[SparkException](df.write.bucketBy(3, "i").format("text").saveAsTable("tt"))
   }
 
-  test("write bucketed data to non-hive-table or existing hive table") {
+  test("write bucketed data using save() or insertInto()") {
```

Comment:
Ideally each test case could be atomic. You could split this test case into two parts.